README.md

```diff
@@ -9,6 +9,5 @@
 | 0 | Ава   | Три идиота                             |
 | 1 | Катя  | Такси                                  |                          
```

Comment:
Almost there! But you are not the first. Please add yourself to the end of the list :)